_examples/digitalidentity/README.md

```diff
@@ -44,3 +44,96 @@ Keeping your settings and access keys outside your repository is highly recommen
 5. Start the compiled program by running `./digitalidentity`
 
 Visiting `https://localhost:8080/` should show a webpage with a Yoti button rendered on it
+
+## Estimated Age Examples
```

Comment:
I don't think we need this additional information in the README, unless we plan to add it for all of the other methods in the SDK